Build failure for tss-esapi 6.1.0 with zeroize_derive 1.2.0

Recently, zeroize_derive 1.2.0 has been released and it causes a compile failure of tss-esapi 6.1.0:

```
   Compiling tss-esapi v6.1.0 (/home/rob/rust-tss-esapi/tss-esapi)
error[E0509]: cannot move out of type `SignatureData`, which implements the `Drop` trait
   --> tss-esapi/src/utils/mod.rs:855:39
    |
855 |                 let signature = match sig.signature {
    |                                       ^^^^^^^^^^^^^ cannot move out of here
856 |                     SignatureData::RsaSignature(signature) => signature,
    |                                                 ---------
    |                                                 |
    |                                                 data moved here
    |                                                 move occurs because `signature` has type `Vec<u8>`, which does not implement the `Copy` trait

error[E0509]: cannot move out of type `SignatureData`, which implements the `Drop` trait
   --> tss-esapi/src/utils/mod.rs:883:39
    |
883 |                 let signature = match sig.signature {
    |                                       ^^^^^^^^^^^^^ cannot move out of here
884 |                     SignatureData::RsaSignature(signature) => signature,
    |                                                 ---------
    |                                                 |
    |                                                 data moved here
    |                                                 move occurs because `signature` has type `Vec<u8>`, which does not implement the `Copy` trait

error[E0509]: cannot move out of type `SignatureData`, which implements the `Drop` trait
   --> tss-esapi/src/utils/mod.rs:911:39
    |
911 |                 let signature = match sig.signature {
    |                                       ^^^^^^^^^^^^^ cannot move out of here
912 |                     SignatureData::EcdsaSignature { r, s } => (r, s),
    |                                                     -  - ...and here
    |                                                     |
    |                                                     data moved here
    |
    = note: move occurs because these variables have types that don't implement the `Copy` trait

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0509`.
error: could not compile `tss-esapi`

To learn more, run the command again with --verbose.
```

The reason for the issue is that previously the Drop trait wasn't being derived for enum types (https://github.com/iqlusioninc/crates/issues/876) and it now is, and because of the severity of problem affected versions of the crate have been yanked.

The changes to this code in 7.0.0-alpha.1 mean that it is unaffected by the problem.

I'm not ready to adapt my code to the API changes in 7.0.0-alpha.1, so would it be possible to create a 6.x.y branch where I could raise a PR to fix this?

--- a/tss-esapi/src/utils/mod.rs
+++ b/tss-esapi/src/utils/mod.rs
@@ -852,7 +852,7 @@ impl TryFrom<Signature> for TPMT_SIGNATURE {
         }
         match sig.scheme {
             AsymSchemeUnion::RSASSA(hash_alg) => {
-                let signature = match sig.signature {
+                let signature = match &sig.signature {
                     SignatureData::RsaSignature(signature) => signature,
                     SignatureData::EcdsaSignature { .. } => {
                         return Err(Error::local_error(WrapperErrorKind::InconsistentParams))
@@ -880,7 +880,7 @@ impl TryFrom<Signature> for TPMT_SIGNATURE {
                 })
             }
             AsymSchemeUnion::RSAPSS(hash_alg) => {
-                let signature = match sig.signature {
+                let signature = match &sig.signature {
                     SignatureData::RsaSignature(signature) => signature,
                     SignatureData::EcdsaSignature { .. } => {
                         return Err(Error::local_error(WrapperErrorKind::InconsistentParams))
@@ -908,7 +908,7 @@ impl TryFrom<Signature> for TPMT_SIGNATURE {
                 })
             }
             AsymSchemeUnion::ECDSA(hash_alg) => {
-                let signature = match sig.signature {
+                let signature = match &sig.signature {
                     SignatureData::EcdsaSignature { r, s } => (r, s),
                     SignatureData::RsaSignature(_) => {
                         return Err(Error::local_error(WrapperErrorKind::InconsistentParams))

